packages/base/src/config/Theme.ts

```diff
@@ -61,11 +63,16 @@ const isTheme = (theme: string) => {
 
 /**
  * Returns if the current theme is part of given theme family.
+ * <b>Note</b>: checks the base theme when custom theme, built via the ThemeDesigner, is set.
+ *
  * @private
  * @param {string} theme the theme family
  * @returns {boolean}
  */
 const isThemeFamily = (theme: string) => {
+	if (!ThemeFamilies.includes(theme)) {
```

Comment:
Could be done in one line:
```ts
getTheme().startsWith(theme) || !!getThemeDesignerTheme()?.baseThemeName?.startsWith(theme)
```

but `getThemeDesignerTheme` performs additional code to check for custom theme and as `isThemeFamily` is called many times (on every icon rendering), preferred to call it only if the current theme is not part of the built-in theme families